var/spack/repos/builtin/packages/cub/package.py

```diff
@@ -34,6 +34,7 @@ class Cub(Package):
     url      = "https://github.com/NVlabs/cub/archive/1.6.4.zip"
 
     version('1.6.4', '924fc12c0efb17264c3ad2d611ed1c51')
+    version('1.4.1', '74a36eb84e5b5f0bf54aa3df39f660b2')
 
     def install(self, spec, prefix):
-        copy_tree('cub', prefix.include)
+        copy_tree('.', prefix)
```

Comment:
Oh, and more importantly, if you have a package that depends on cub, the Spack compiler wrappers will automatically inject `-I/path/to/cub/prefix/include` on the link line. So it's definitely best to do it this way. Thanks!